src/features/utils/aggregate-base.js

```diff
@@ -49,7 +50,9 @@ export class AggregateBase extends FeatureBase {
   checkConfiguration () {
     // NOTE: This check has to happen at aggregator load time
     if (!isConfigured(this.agentIdentifier)) {
-      let jsAttributes = { ...gosCDN().info?.jsAttributes }
+      const cdn = gosCDN()
+      if (!cdn.info?.licenseKey || !cdn.info?.applicationID) return warn(43)
```

Comment:
We may need to abort the agent/eventemitter and flush the backlog so we are no longer capturing/retaining data.

Reply:
the feature aggs are going to continue initializing so shouldn't flush backlog

Reply:
I think it should flush and abort the EE.  The feature aggs will never drain because waitForFlags will never fire